config/default.py

```diff
@@ -37,6 +37,7 @@ class Default:
     INIT_VERTEX: bool = True
 
     GENMEDIA_BUCKET = os.environ.get("GENMEDIA_BUCKET")
+    GENMEDIA_GENERATED_BUCKET = os.environ.get("GENMEDIA_GENERATED_BUCKET")
```

Comment:
What is this new env variable?

Reply:
@mayureshagashe2105 this was the bucket I used for the SD generated images that would be used for the study. I can remove

Reply:
Removed.